CMakeLists.txt

```diff
@@ -1245,16 +1245,10 @@ if(VRPN_BUILD_SERVER_LIBRARY)
 	if(VRPN_INSTALL)
 		install(TARGETS
 			vrpnserver
-			ARCHIVE
-			DESTINATION
-			lib
-			COMPONENT
-			serversdk
-			PUBLIC_HEADER
-			DESTINATION
-			include
-			COMPONENT
-			serversdk)
+			RUNTIME DESTINATION bin COMPONENT serversdk
+			LIBRARY DESTINATION lib${LIB_SUFFIX} COMPONENT serversdk
+			ARCHIVE DESTINATION lib${LIB_SUFFIX} COMPONENT serversdk
```

Comment:
This is good, but I'm wondering if it makes more sense to (bump the CMake version if necessary) and use GNUInstallDirs - it's what I've done with all my more recent projects, and ends up nicely supporting even, for example, Debian multi-arch. I'd link to my blog post but apparently my wordpress install is broken... :anguished: